pkgs/applications/misc/qMasterPassword/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, qt5, libX11, libXtst, openssl, libscrypt }:
+
+stdenv.mkDerivation rec {
+  name = "qMasterPassword";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "bkueng";
+    repo = name;
+    rev = "v${version}";
+    sha256 = "0l0jarvfdc69rcjl2wa0ixq8gp3fmjsy9n84m38sxf3n9j2bh13c";
+  };
+
+  buildInputs = [ qt5.qtbase libX11 libXtst openssl libscrypt ];
+  nativeBuildInputs = [ qt5.qmake ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/{applications,pixmaps,icons/qmasterpassword,doc/qMasterPassword}
+    mv qMasterPassword $out/bin
+    mv data/qMasterPassword.desktop $out/share/applications
+    mv data/icons/app_icon.png $out/share/pixmaps/qmasterpassword.png
+    mv LICENSE README.md $out/share/doc/qMasterPassword
+    mv data/icons/* $out/share/icons/qmasterpassword
+  '';
+
+  meta = with stdenv.lib; {
+      description = "Stateless Master Password Manager";
+      homepage = https://github.com/bkueng/qMasterPassword;
+      license = licenses.gpl3;
+      maintainers = [ maintainers.tadeokondrak ];
```

Comment:
You need to define `meta.platforms` (https://nixos.org/nixpkgs/manual/#chap-meta)